### DISCUSSION TITLE
Error in the documention for the wire format or did i get it wrong?

### DISCUSSION THREAD

**Discussion by thatv** [markqvist/Reticulum#148] [Help & Questions] [answered]
Hi there, currently i am looking at the doc of the project and trying to understand the wire format. I am not sure if i get something wrong or I have found an error in the section with the Packet Examples. The second and third example stating a Hop count of 00000111 which is stated as zero hops. shouldn't it stated as 7 hops?

+- Packet Example -+

   HEADER FIELD   DESTINATION FIELD   CONTEXT FIELD  DATA FIELD
 _______|_______   _______|_______   ________|______   __|_
|               | |               | |               | |    |
00000000 00000111 [HASH1, 16 bytes] [CONTEXT, 1 byte] [DATA]
|| | | |    |
|| | | |    +-- Hops             = 0
|| | | +------- Packet Type      = DATA
|| | +--------- Destination Type = SINGLE
|| +----------- Propagation Type = BROADCAST
|+------------- Header Type      = HEADER_1 (two byte header, one address field)
+-------------- Access Codes     = DISABLED

Thanks for the Clarification. And a nice Project btw.

**Comment by markqvist** [comment, marked as answer]
No, that is simply a typo on my part. Must have set the actual bit fields and then forgotten to update the explanation ;) Thanks for pointing it out, I will fix it!
